samples/kfp-user-guide/README.md

```diff
@@ -23,20 +23,20 @@ def echo_pipeline(
 ```
 ## Table of Contents
 
-  - [Pipeline compilation](#pipeline-compilation)
+  - [Compile Pipelines](#compile-pipelines)
     - [1. Compile pipelines using the `kfp_tekton.compiler.TektonCompiler` in Python](#1-compile-pipelines-using-the-kfp_tektoncompilertektoncompiler-in-python)
     - [2. Compile pipelines using the `dsl-compile-tekton` bash command line tool](#2-compile-pipelines-using-the-dsl-compile-tekton-bash-command-line-tool)
-  - [Uploading pipelines](#uploading-pipelines)
+  - [Upload Pipelines](#upload-pipelines)
     - [1. Upload pipelines using the Kubeflow Pipeline user interface.](#1-upload-pipelines-using-the-kubeflow-pipeline-user-interface)
     - [2. Upload pipelines using the `kfp_tekton.TektonClient` in Python](#2-upload-pipelines-using-the-kfp_tektontektonclient-in-python)
     - [3. Upload pipelines using the `kfp` bash command line tool](#3-upload-pipelines-using-the-kfp-bash-command-line-tool)
-  - [Pipeline execution](#pipeline-execution)
+  - [Execute Pipelines](#execute-pipelines)
```

Comment:
IMO the word "execute" conjures up images of people getting their life cut short involuntarily

Could we use "run" and "running" a pipeline instead?

Reply:
We can change the sub-header to run if it sounds bad. I just felt a bit repetitive saying "run" so many times in this doc when I say run this command to run some action. In kfp although their api is "run", but they do use a mix of "run" and "execute" in some part of their docs to make it sound less repetitive.